### DISCUSSION TITLE
SDRTrunk and rdio-scanner

### DISCUSSION THREAD

**Discussion by johnmac78** [chuot/rdio-scanner#282] [Q&A]
I am trying to set up SDRTrunk with rdio-scanner and have not been able to get it to work. I have configured the following -

1. dirwatch directory
2. No downstream
3. Options - No AFS systems (No idea what that is)
4. Systems - see attached  SDRTrunk channels screen (No idea what to put in ID field) System field is exactly what is in SDRTrunk
5. Talkgroups have been downloaded and converted from RadioReference.
Any suggestions would be appreciated.

This is the system I am monitoring with SDRTrunk     [https://www.radioreference.com/db/site/26397](url)

Thank you,
John

![Screenshot 2022-11-30 152448](https://user-images.githubusercontent.com/117118297/204900879-3f68772b-7824-460d-920d-9f1cbc3dab74.jpg)

**Comment by chuot** [comment]
You need to define the talkgroups you want to save to file by creating an alias list, which seems to be already defined (Worcester) for your system. So go to the *Alias* tab and specify which talkgroup should be recorded. Then Rdio Scanner should pick them up based on the dirwatch you set.

**Reply by johnmac78** [reply]
I have some talkgroups set up for record, but it does not seem to activate rdio-scanner. still need to know what to put in the ID tab in the config file. Calls are recording properly from SDRTrunk.

Thanks for the reply
John

**Reply by chuot** [reply]
I'm a bit confuse about the ID tab in the config; which config?.

Can you post a screenshot of your dirwatch definition that is monitoring the recordings folder from sdr-trunk?

**Reply by johnmac78** [reply]
All set - The dirwatch was in the disabled position. Receiving talkgroups now

John

**Reply by chuot** [reply]
That is good news. Happy Rdio Scanning!
